roles/ipabackup/library/ipabackup_get_backup_dir.py

```diff
@@ -61,7 +61,7 @@
 
 def main():
     module = AnsibleModule(
-        argument_spec=dict(),
+        argument_spec={},
```

Comment:
Is this change needed for pylint?

Reply:
Yes, it is needed. An alternative is to disable this check.

Reply:
So, our pre-commit hook uses pylint 2.10.2, but Fedora 35 provides 2.9.6 (latest upstream is 2.12.2).

If we stick to Fedora's version, this change is not needed. It will only become necessary when 2.10 series land on Fedora.